Phindr3D-Python/src/Data/ImageLayer.py

```diff
@@ -14,15 +14,35 @@
 # You should have received a copy of the GNU General Public License
 # along with src.  If not, see <http://www.gnu.org/licenses/>.
 
+from ImageChannel import *
+
 class ImageLayer:
     """This class handles groups of image files and the associated metadata.
        Static methods that draw closely from transliterations of the MATLAB functions
        can be found in the DataFunctions class."""
 
     def __init__(self):
         """ImageChannel class constructor"""
+        self.well = None
+        self.field = None
+        self.channels = {}
         pass
 
+    def setWell(self, well):
+        self.well = well
+
+    def setField(self, field):
+        self.field = field
+
+    def addChannels(self, channels): # channels is a tuple/line of metadata
+        count = 1
+        for index in range(3):
```

Comment:
This assumes there will always be 3 channels. Can you get the range from the length of channels?